views/pbxsManager.html

```diff
@@ -30,6 +30,13 @@
 				</div>
 			</div>
 
+			<div id="unassigned_select_all" class="row-fluid">
+				<div class="span12 text-center link-box">
+					<i class="fa fa-check"></i> <span class="unassigned_select_all_label">{{ i18n.select_all }}</span>
```

Comment:
`class` attributes should be lowercased, each word separated by a dash, as opposed to `id` attributes that should be lowercased, each word separated by an underscore ([rule](https://docs.2600hz.com/ui/docs/codingStandards/#naming)).
```suggestion
					<i class="fa fa-check"></i> <span class="unassigned-select-all-label">{{ i18n.select_all }}</span>
```

Reply:
Done, thanks!